apps/ledger-live-desktop/src/renderer/components/TranslatedError/hooks/useErrorLinks.tsx

```diff
@@ -17,9 +17,9 @@ export function useErrorLinks(error?: Error | null) {
   const history = useHistory();
   return useMemo(() => {
     if (!!error && "links" in error && Array.isArray(error.links)) {
-      const safeStringLinks = error.links.filter(
-        (link): link is string => typeof link === "string",
-      );
+      const safeStringLinks = error.links
+        .filter((link): link is string => typeof link === "string")
+        .map(link => link);
```

Comment:
i'm not sure to see what the code change bring?

Reply:
Absolutely nothing 😄 WIll remove.